NYPLCardCreator/Classes/AlternativeAddressesViewController.swift

```diff
@@ -32,16 +32,31 @@ final class AlternativeAddressesViewController: TableViewController {
   
   override func viewDidLoad() {
     super.viewDidLoad()
+    
+    let headerView = UIView(frame: CGRect(x: 0, y: 0, width: self.view!.bounds.size.width, height: 80.0))
```

Comment:
All of the sizes of views in the card creator were previously set without any fixed numbers like `80.0`. This is important for supporting things like dynamic text sizing and text of different lengths depending on the current locale. I would rework this so you do not need to rely on a magic value for the size of the `UIView`. If you need some advice on how to do this, please poke me!